core/transaction_pool/clock.hpp

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_CLOCK_HPP
+#define KAGOME_CLOCK_HPP
+
+#include <chrono>
+
+namespace kagome::transaction_pool {
+
+  /**
+   * An interface for a clock
+   */
+  class Clock {
+   public:
+    /**
+     * Difference between two time points
+     */
+    using Duration = std::chrono::milliseconds;
```

Comment:
use 
```
using ClockType = std::chrono::steady_clock;
using Duration = ClockType::duration;
using TimePoint = ClockType::time_point;
```

Reply:
Hm, I actually wanted to avoid sticking to a particular clock type, but it seems like the steady clock is the only one fit anyway.

Reply:
You can make Clock generic